internal/datastore/mysql/datastore_test.go

```diff
@@ -94,13 +94,6 @@ func TestMySQLDatastoreDSNWithoutParseTime(t *testing.T) {
 	require.ErrorContains(t, err, "https://spicedb.dev/d/parse-time-mysql")
 }
 
-func TestMySQL5Datastore(t *testing.T) {
-	b := testdatastore.RunMySQLForTesting(t, "")
-	dst := datastoreTester{b: b, t: t}
-	test.All(t, test.DatastoreTesterFunc(dst.createDatastore))
-	additionalMySQLTests(t, b)
-}
-
 func TestMySQL8Datastore(t *testing.T) {
```

Comment:
https://github.com/josephschorr/spicedb/blob/608f9eccb7ec17b63882f985f5ee90fb8f6d5649/internal/testserver/datastore/mysql.go#L37

I think you want to remove a bunch of the v5 stuff here as well

Reply:
Looks like the file forgot to save; VSCode does that occasionally